hail/python/hailtop/utils/utils.py

```diff
@@ -184,10 +184,14 @@ def is_transient_error(e):
 
 async def request_retry_transient_errors(session, method, url, **kwargs):
     delay = 0.1
+    errors = 0
     while True:
         try:
             return await session.request(method, url, **kwargs)
         except Exception as e:
+            errors += 1
+            if errors % 10 == 0:
+                log.warning(f'encountered 10 errors, most recent one was {e}', exc_info=True)
```

Comment:
should this be `encountered {errors} errors`? might be useful to know how many exceptions are getting thrown

Reply:
Isn't how many exceptions get thrown number of error messages * 10? If we have a bunch of cumulative sums, and they come from different retries, it might be harder to tell how many total there were.

Reply:
could give each request a UID and print that too?

Reply:
the cumulative sum is per request, though. So the number printed will be the cumulative sum for a given request, so if you see 5000 errors, that means one request had 5k errors, not 5k requests with one.